### DISCUSSION TITLE
Kernel selection for NS equation

### DISCUSSION THREAD

**Discussion by Wolke926** [idaholab/moose#24904] [Q&A Modules: Navier-Stokes]
Dear MOOSE experts:
 When I try to combine the Navier-Stokes equation and the Allen-Cahn equation, there is a surface tension term(marked in equation 2) and I don't know how to choose a suitable kernel for this term.
    Comparing the form of the Cahn-Hilliard equation, the surface tension term in equation 2 looks similar to equation 1. The functional derivative of concentration can be regarded as chemical potential, and the difference between Cahn Hilliard equation and the surface tension term is that the former is \nebla (\M*\nebla miu) while in equation 2 is \phi*\ nebla miu (although strictly speaking, only the functional derivative of energy with respect to concentration can be called chemical potential).

   I wonder if I can still use the SplitCHParsed kernel (although )with some other kernels to solve this term or should I write a new one?

![Screen Shot 2023-07-05 at 10 50 54 AM](https://github.com/idaholab/moose/assets/120539827/dffaa149-5b52-44be-987c-668459d90fa0)

**Comment by lindsayad** [comment]
@dschwen any thoughts on this?

**Comment by Wolke926** [comment]
Does anybody have some ideas?
